src/main/webapp/js/courseSelector.js

```diff
@@ -23,21 +23,25 @@ export const CollegePlanner = (() => {
    * Gets departments from /api/departments servlet to populate dropdown list
    */
   async function getDepartmentOptions() {
-    const response = await fetch('/api/departments');
-    const departmentList = await response.json();
-    const departmentContainer = document.getElementById('departments');
-    departmentContainer.innerHTML =
-        '';  // Clearing departmentContainer to get rid of previous options
-    // Add instruction to courseList dropdown
-    const option = document.createElement('option');
-    option.innerText = 'Select a Department';
-    option.selected = true;
-    option.hidden = true;
-    departmentContainer.appendChild(option);
-    // Add each course to course list
-    const departmentsDetailed = departmentList.departments;
-    departmentsDetailed.forEach(
-        department => addDepartmentOption(department, departmentContainer));
+    try {
+      const response = await fetch('/api/departments');
+      const departmentList = await response.json();
```

Comment:
Just put the `await` lines inside the try catch? The other lines below these won't cause an error. And having everything inside the try catch causes another level of indentation